modules/virt-deprecated-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-deprecated-tasks_{context}"]
+= Removing deprecated or unused resources
+
+* If you no longer use Tekton (TTO) in your environment after upgrading to OpenShift Container Platform 4.14 or later, you can remove any remaining Tekton-related resources that were previously managed by the ssp-operator.
+
+.Procedure
+
+. Use the following command to remove any leftover Tekton-related resources from the cluster:
+
+[source,terminal]
+----
+oc delete clusterroles,rolebindings,serviceaccounts,configmaps,pipelines,tasks \
+  --selector 'app.kubernetes.io/managed-by=ssp-operator' \
+  --selector 'app.kubernetes.io/component in (tektonPipelines,tektonTasks)' \
+  --selector 'app.kubernetes.io/name in (tekton-pipelines,tekton-tasks)' \
+  --ignore-not-found \
+  --all-namespaces
+----
+. If the Tekton custom resource definitions (CRDs) have already been removed, the command might return an error. You can safely ignore this error. All other matching resources will still be deleted.
```

Comment:
- If/then
- remove "still" (minimalism)
- change leftover to remaining

Recommend:
If the Tekton custom resource definitions (CRDs) have already been removed, then the command might return an error. You can safely ignore this error. All other matching resources will be deleted.